lib/pc_api.h

```diff
@@ -126,7 +126,7 @@ typedef struct
 
 typedef struct
 {
-	int8_t readonly;
+	void *mem; /* An opaque memory buffer to be freed on destruction if not NULL */
```

Comment:
I would remove the comment here, as it pertains to the internals and not the API.